static/app/components/events/featureFlags/utils.tsx

```diff
@@ -139,3 +142,11 @@ export const PROVIDER_TO_SETUP_WEBHOOK_URL: Record<WebhookProviderEnum, string>
   [WebhookProviderEnum.UNLEASH]:
     'https://docs.sentry.io/organization/integrations/feature-flag/unleash/#set-up-change-tracking',
 };
+
+export function shouldShowFeatureFlagCTA(organization: Organization, project: Project) {
+  return (
+    !project.hasFlags &&
+    featureFlagOnboardingPlatforms.includes(project.platform ?? 'other') &&
+    organization.features.includes('feature-flag-cta')
```

Comment:
we probably don't need a specific feature-flag for the cta. the whole flyout is under a FF already, and if `project.hasFlags` is not true then we should be showing setup instructions at all times imo